### ISSUE TITLE
ShareX screenshots are categorized as Photos not Screenshots

### ISSUE THREAD

**Issue by cjmling** [spacedriveapp/spacedrive#1527]
### Describe the bug

I use ShareX to capture screenshots on my Windows. Upon viewing in Overview tab , these files are categorized into Photos rather than Screenshots

![image](https://github.com/spacedriveapp/spacedrive/assets/1612187/28e87bf4-be76-459f-95f3-b4e5c794bc49)

### Reproduction

1. Go to "Overview"

### Expected behavior

These small images should be categorized into "Screenshots"

### Platform and versions

```Shell
v0.1.0 - fdc3c30
Windows 11
```

**Comment by brxken128** [comment]
Indeed this is a bug, but I don't think we have the features in place to differentiate between the two. If the extension matches that of an image, it's an image. Differentiating between them isn't too trivial, either - but it will be looked at.

**Comment by Brendonovich** [comment]
Yeah 'Screenshots' don't actually do anything right now, and tbh i'm not sure it should exist. Idk how we'd detect them.

**Comment by jamiepine** [comment]
Closing due to v2 rewrite. If you encounter this issue in v2, please open a new issue.
